README.md

```diff
@@ -1,37 +1,18 @@
-# cfn-pwm
+## cfn-pwm
 PWM (https://github.com/pwm-project/pwm) is an open source password self service application for LDAP directories.
 This cloudformation stack creates the infrastructure needed to run PWM behind an application load balancer
 
-# Templates
-## Compound Template
-The root template that forms the basis of the remaining templates
+## Prequisites
+* S3 bucket with the following configuration files
+  * `PwmConfiguration.xml` - the config file used to configure PWM
```

Comment:
I know we have our implementation specific artifacts to fall back on for reference but thoughts on including a bare bones example on how these would be structured (or a link to an external reference if the config is standardized). 

Same question for the other required resources

Reply:
I do think a future implementation should include a bare bones configuration file. We just aren't there yet.

Reply:
Can we point to the PWM docs as an example? Same for the war files, where can a user get that?